### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/vanilla-extract/commit/6ade89a86560df6019ae34f217cd28810b5bf80d#commitcomment-51789331

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/vanilla-extract/commit/cfe5b0c1e010df50efba70fc1d532745c2ff8356

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: site/src/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -187,7 +187,7 @@ export const HomePage = () => {
                 </Box>
                 <Code
                   language="tsx"
-                  errorTokens={['brandd', 'large']}
```
[the remark this answers is not in the document]
Comment (michaeltaranto) [reply site/src/HomePage/HomePage.tsx:190]: Yeah these are all deliberate errors in this file. Used on [the site](https://vanilla-extract.style/) to highlight typos/spelling mistakes.

File: site/src/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -187,7 +187,7 @@ export const HomePage = () => {
...
+                  errorTokens={['brand_', 'large']}
```
Comment (jsoref) [inline site/src/HomePage/HomePage.tsx:190]: I presume that `brand_` is just as good for highlighting that.

That said, if you want to keep these spellings, I'm happy to drop these changes.

Comment (michaeltaranto) [reply site/src/HomePage/HomePage.tsx:190]: I get they are both incorrect values yeah. Personally though i'd prefer to stick the the "typo" that's in there.

Comment (jsoref) [reply site/src/HomePage/HomePage.tsx]: Dropped

**Review by michaeltaranto — APPROVED** [review]

**Comment by michaeltaranto** [comment]
Thanks for the corrections 😉

**Comment by jsoref** [comment]
You're welcome. If you'd like to use the action to prevent additional _unintentional_ typos from creeping in, let me know. It's pretty easy to tell it that certain typos are intentional, and then it'll leave them be 🙂 .

**Pull request merged**